spec/javascripts/health-care/components/social-security-number.spec.jsx

```diff
@@ -1,75 +1,63 @@
 import React from 'react';
-import ReactTestUtils from 'react-addons-test-utils';
+import SkinDeep from 'skin-deep';
+
 import SocialSecurityNumber from '../../../../_health-care/_js/_components/social-security-number';
 
 describe('<SocialSecurityNumber>', () => {
-  let component = null;
-
-  beforeEach(() => {
-    component = ReactTestUtils.renderIntoDocument(
-      <SocialSecurityNumber ssn={{ ssn: '999-99-9999' }} onUserInput={(_update) => {}}/>
-    );
-    assert.ok(component, 'Cannot even render component');
+  describe('propTypes', () => {
+    let consoleStub;
+    beforeEach(() => {
+      consoleStub = sinon.stub(console, 'error');
+    });
+
+    afterEach(() => {
+      consoleStub.restore();
+    });
+
+    it('ssn is required', () => {
+      SkinDeep.shallowRender(<SocialSecurityNumber/>);
+      sinon.assert.calledWithMatch(consoleStub, /Required prop `ssn` was not specified in `SocialSecurityNumber`/);
+    });
+
+    it('ssn must be a string', () => {
+      SkinDeep.shallowRender(<SocialSecurityNumber ssn/>);
+      sinon.assert.calledWithMatch(consoleStub, /Invalid prop `ssn` of type `boolean` supplied to `SocialSecurityNumber`, expected `string`/);
+    });
+
+    // TODO(awong): Why in the world does this not work?!?!
```

Comment:
This is very strange, I'm trying to run this locally too and have no idea why it's failing...